SimCalorimetry/HcalSimProducers/python/hcalSimParameters_cfi.py

```diff
@@ -114,16 +114,26 @@
 
 hcalSimParameters.hoHamamatsu = hcalSimParameters.ho.clone()
 
-# Customises the HCal digitiser for post LS1 running
+# Customises the HCAL digitiser for post LS1 running
 from Configuration.Eras.Modifier_run2_common_cff import run2_common
+from Configuration.ProcessModifiers.applyHFLibraryFix_cff import applyHFLibraryFix
+
 run2_common.toModify( hcalSimParameters, 
-    ho = dict(
-        siPMCode = cms.int32(1)
-    ),
-    hf1 = dict( samplingFactor = cms.double(0.335) ),
-    hf2 = dict( samplingFactor = cms.double(0.335) )
+    ho = dict( siPMCode = 1 ),
+#--- Default for Run2 HFShowerLibrary file
+    hf1 = dict( samplingFactor = 0.335 ),
+    hf2 = dict( samplingFactor = 0.335 )
 )
-
+#--- Alternative: usage of Run3 HFShowerLibrary file for Run2
+(applyHFLibraryFix & run2_common).toModify( hcalSimParameters, 
+    hf1 = dict( samplingFactor = 0.37,
+                timePhase = 9.0
+              ),
+    hf2 = dict( samplingFactor = 0.37,
+                timePhase = 8.0
+              )
+)
```

Comment:
On the other hand, in here these customizations made with `applyHFLibraryFix` are also done as part of customizations made with `run3_common` modifier further below.

While this arrangement works technically, I'm concerned at some point this "duplication" would cause confusion. An alternative would be to make the necessary customizations only with `run3_HFSL` modifier. In this way the fix could be enabled with `cmsDriver.py`'s `--eras` parameter along`--eras Run2_2018,run3_HFSL`. A downside is that the purpose of using `run3_HFSL` in this way would be less clear without additional documentation.